docs/advanced_reference/commands/autonomy_deploy.md

```diff
@@ -157,6 +163,37 @@ export OPEN_AUTONOMY_PRIVATE_KEY_PASSWORD=PASSWORD
 autonomy deploy build (...)
 ```
 
+### Customize resources
+
+You can customize the resource values for the agent nodes using `--agent-cpu` and `--agent-memory` flags or using `AUTONOMY_AGENT_CPU` and `AUTONOMY_AGENT_MEMORY` environment variables.
+
+```bash
+autonomy deploy build --agent-cpu 1.0 --agent-memory 2048
+```
+
+will produce agent nodes with following resource constrains
+
+```yaml
+  xyz_abci_0:
+    mem_reservation: 256M
```

Comment:
How does one specify the reservation/requests? Seems like this PR should also allow for this.